static/src/stylesheets/module/onward/_feedback.scss

```diff
@@ -0,0 +1,55 @@
+
+.feedback__form {
+    border: 0px;
+    outline: none;
+    display: none;
+}
+
+.feedback__form--selected {
+    display: block;
+}
+
+.feedback__element {
+    margin-bottom: 1.5rem;
+    margin-top: 1.5rem;
```

Comment:
For `padding` and `margin`, we like to use multiples of `$gs-baseline` (for vertical spacing) and `$gs-gutter` (for horizontal spacing)

[Example](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/atoms/_quiz.scss#L171)
[More extreme example](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/atoms/_quiz.scss#L286)